### DISCUSSION TITLE
BertViz for custom models

### DISCUSSION THREAD

**Discussion by Sanbot-okk** [jessevig/bertviz#109] [Q&A]
Is it possible to visualize attention with BertViz for custom transformer models?

**Comment by jessevig** [comment]
Hi @Sanbot-okk do you mean custom weights, or custom architecture? If you use a Huggingface model with custom weights, it should just work (except for the neuron view, which only works for a select set of pretrained models).  You would need to set `output_attentions` as in the example scripts, and initialize the model from your own local directory. Let me know if this is the case, or if you are trying to do something else.
